versions.txt

```diff
@@ -4,8 +4,8 @@
 google-cloud-firestore:3.0.9:3.0.10-SNAPSHOT
 google-cloud-firestore-admin:3.0.9:3.0.10-SNAPSHOT
 google-cloud-firestore-bom:3.0.9:3.0.10-SNAPSHOT
-google-cloud-firestore-parent:3.0.9:3.0.10-SNAPSHOT
```

Comment:
@Neenu1995 can you take a look why this was deleted?

Reply:
the parent version is same as the google-cloud-library version. Doesn't have to be tracked on versions.txt